tests/foreman/api/test_repository_set.py

```diff
@@ -9,9 +9,9 @@
 
 :CaseAutomation: Automated
 
-:CaseLevel: Acceptance
+:CaseLevel: Component
 
-:CaseComponent: API
+:CaseComponent: Repositories
```

Comment:
Not sure if that valid component `:CaseComponent: Repositories` already has cover, don't we have  Repository_Set

@mirzal Is the testimony.yaml is single truth or we can add Components similar like this case?

Reply:
@omkarkhatavkar we do not have any component name on Bugzilla with name `Repository/Repositories Set` it's only `Repositories`

Reply:
ok then

Reply:
@omkarkhatavkar Bugzilla is our source of truth. `testimony.yaml` is created automatically by extracting values from Bugzilla. If you see any discrepancy between Bugzilla and `testimony.yaml`, then trust Bugzilla - and let me know, `testimony.yaml` should be fixed.